### DISCUSSION TITLE
How to Restore Context Menu Registry

### DISCUSSION THREAD

**Discussion by kumodot** [Tichau/FileConverter#447] [Q&A]
For some reasone i was fixing some permissions on a second SSD by using commands like takedown, icacls, etc and somewhat they screwed up some permissions on my windows 11 for other apps installed. Now my file Converter is installed, but it does not show the context menu. 
   Already tried reinstalling it (not from scratch, just repair) and it didn't worked. 
      Is there a way to manually fix the registry to have it working again ? 
          I will try to remove it totally and install from scratch now to see if works.

**Comment by kumodot** [comment]
I got it fixed ! Uninstalled and deleted all references on registry manually, installed from scratch and its fixed !

**Reply by alienmau** [reply]
dont work, in the version 1.2.3 appears in the menu context, but when i upgraded to 2.0.2 doesnt work, i did complete unnistall, delete any registry but do not work, any other solution?

**Reply by veloursatellite** [reply]
I have the same problem with 2.0.2 version. Did you find any solution yet?
